island-is/island.is
# feat(samradsgatt): Add breadcrumbs to the about page

# Add breadcrumbs to the about page

## What

Add breadcrumbs to the about page

## Why

Is needed.

## Screenshots / Gifs

<img width="1045" alt="Screenshot 2023-01-16 at 10 33 00" src="https://user-images.githubusercontent.com/106164521/212657849-3b173600-ae53-49c6-b270-a85b1afeeeb9.png">

The figma desing is not correct: https://www.figma.com/file/gFSUsFCIMYWRGa94XsmhpT/DEV-Samr%C3%A1%C3%B0sg%C3%A1tt?node-id=209%3A8016&t=TMSB3vYqcaBWfSNr-0

## How to test

Checkout branch feature/samradsgatt/breadcrumbs-a-about-sidu
Run samradsgatt locally: yarn start samradsgatt
Navigate to http://localhost:4200/samradsgatt/About
Look at the breadcrumbs in the top left of the page
Should look similar the figma design

## Checklist:
